int32 id
string[] transitions
---
bool status